hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/FederationInterceptorREST.java

```diff
@@ -586,37 +583,31 @@ private Response invokeSubmitApplication(ApplicationSubmissionContextInfo submis
    * operation.
    */
   @Override
-  public AppInfo getApp(HttpServletRequest hsr, String appId,
-      Set<String> unselectedFields) {
+  public AppInfo getApp(HttpServletRequest hsr, String appId, Set<String> unselectedFields) {
 
     long startTime = clock.getTime();
 
-    ApplicationId applicationId = null;
     try {
-      applicationId = ApplicationId.fromString(appId);
+      ApplicationId.fromString(appId);
```

Comment:
Why do we do this if we don't use it?

Reply:
Thanks for the reminder, I will improve this part of the code again.